builtin/providers/openstack/provider.go

```diff
@@ -147,6 +147,7 @@ func Provider() terraform.ResourceProvider {
 			"openstack_fw_firewall_v1":                 resourceFWFirewallV1(),
 			"openstack_fw_policy_v1":                   resourceFWPolicyV1(),
 			"openstack_fw_rule_v1":                     resourceFWRuleV1(),
+			"openstack_image_v2":                       resourceImageV2(),
```

Comment:
I started working on an Image data source and waffled over what the resource name should be. Currently, the OpenStack resources are named with the following format:

```
openstack_<service>_<resource>_<service api version>
```

With that in mind, I was going to use the following for the Image data source:

```
openstack_images_image_v2
```

`images_image` looks a little goofy, but it makes sense in the context of something like:

```
openstack_images_imagedata_v2
openstack_images_artifact_v2
```

So I'd recommend changing the name to `openstack_images_image_v2` here. Thoughts?